src/components/Menu.js

```diff
@@ -121,6 +130,8 @@ const Avatar = styled.div`
     padding-right: 8px;
     & img {
         width: 28px;
+        border-radius: 100px;
+        object-fit: cover;
     }
```

Comment:
Para que o `object-fit: cover;` funcione de forma apropriada é necessario informar a altura da imagem tambem.
```suggestion
    & img {
        width: 28px;
        height: 28px;
        border-radius: 100px;
        object-fit: cover;
    }
```